spec/abi.dd

```diff
@@ -68,9 +68,14 @@ $(H2 $(LNAME2 delegates, Delegates))
         function's stack frame (nested functions).
         )
 
-$(H2 $(LNAME2 structs, Structs))
+$(H2 $(LNAME2 structs, Structs and Unions))
 
-        $(P Conforms to the target's C ABI struct layout.)
+        $(P These conform to the target's C ABI struct layout, except:)
+
+        * An `extern(D)` struct or union with no fields has size 1, not 0.
```

Comment:
Also `extern(C++)` structs seem to have non-zero size judging by running a simple example. The C++ Standard (all versions) agrees with that.
```suggestion
        * An `extern(D)` or `extern(C++)` struct or union with no fields has size 1, not 0.
```

Reply:
This paragraph is about differences from C. The link that follows is for further information.